### DISCUSSION TITLE
I cannot see the  org.apache.parquet.* logs.

### DISCUSSION THREAD

**Discussion by jiangjiguang** [trinodb/trino#13452] [Q&A]
I have installed trino on a single machine for testing, 
but I only can see the io.trino.*  logs, cannot  see the  org.apache.parquet.* logs.
how can I see the org.apache.parquet.* logs?

the config is:

[root@icx20 etc]# cat log.properties
io.trino=DEBUG
org.apache.parquet=DEBUG

**Comment by findepi** [comment]
the logs are filtered out by how https://github.com/trinodb/trino-hadoop-apache is built
cc @electrum 

perhaps there is an existing issue about this.

**Comment by jiangjiguang** [comment]
Since I have done some optimizations on parquet， and want to check the effect through the org.apache.parquet.*  logs.
Is there any way to open the org.apache.parquet.*  logs ?
